src/course-unit/course-xblock/CourseXblock.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import { Dropdown, Icon, IconButton } from '@edx/paragon';
+import { EditOutline as EditIcon, MoreVert as MoveVertIcon } from '@edx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import messages from './messages';
+
+const CourseXblock = ({ id, title }) => {
+  const intl = useIntl();
+
+  return (
+    <li className="studio-xblock-wrapper mb-4">
+      <section className="wrapper-xblock p-4">
+        <div className="xblock-header d-flex justify-content-between mb-4">
+          <div className="header-details">
+            <span className="h3 xblock-display-name">{title}</span>
+          </div>
+          <div className="header-actions flex-shrink-0">
+            <ul className="actions-list list-unstyled m-0 d-flex">
+              <li className="action-item action-edit">
+                <IconButton
+                  alt={intl.formatMessage(messages.blockAltButtonEdit)}
+                  iconAs={EditIcon}
+                  size="sm"
+                  onClick={() => {}}
+                />
+              </li>
+              <li className="action-item action-actions-menu">
+                <Dropdown>
+                  <Dropdown.Toggle
+                    id={id}
+                    as={IconButton}
+                    src={MoveVertIcon}
+                    alt="unit-xblock-card-header__menu"
```

Comment:
[important] Let's add a more meaningful alternative description for this button and create a translation for it

Reply:
Changed